packages/bitcore-wallet-service/test/integration/server.js

```diff
@@ -8933,6 +8933,7 @@ describe('Wallet service', function() {
           token: 'DEVICE_TOKEN',
           packageName: 'com.wallet',
           platform: 'Android',
+          walletId: '123'
```

Comment:
keep the old tests (As the old clients should still work) and add a new tests with the new `walletId` member.